inc/class-wpseo-utils.php

```diff
@@ -973,6 +973,23 @@ public static function get_user_locale() {
 		return get_locale();
 	}
 
+	/**
+	 * Returns the full name for the user's language.
+	 *
+	 * @since 5.9
+	 *
+	 * @returns string The language name.
+	 */
+	public static function get_language_name() {
```

Comment:
Please make a language util class instead of adding stuff to this class. We are trying to reduce the amount of methods over here.